front/src/core-logic/domain/feedback/feedback.slice.ts

```diff
@@ -23,11 +25,15 @@ const topics = [
   "dashboard-agency-register-user",
   "auth-global",
   "establishments-batch",
+  "search-result-page",
```

Comment:
pour détacher la notion de "page":

```suggestion
  "search-result",
```